.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log
 *.swp
+.devcontainer/
```

Comment:
move to .gitignore global, it should copy to container's .gitignore global (if not, new computer is why)